docs/book/src/cronjob-tutorial/testdata/project/internal/webhook/v1/webhook_suite_test.go

```diff
@@ -67,8 +66,18 @@ var _ = BeforeSuite(func() {
 
 	ctx, cancel = context.WithCancel(context.TODO())
 
+	var err error
+	err = batchv1.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	err = admissionv1.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	// +kubebuilder:scaffold:scheme
+
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
+		CRDInstallOptions:     envtest.CRDInstallOptions{Scheme: scheme.Scheme},
```

Comment:
Why this option was added?
I do not think that we need it by default?

Reply:
it is needed for webhook tests https://github.com/kubernetes-sigs/kubebuilder/pull/4441#discussion_r1898614156

I think only needed for webhook, but would add to both, for consistency